examples/basic/main.tf

```diff
@@ -11,15 +11,18 @@ data "aws_vpc" "main" {
   default = true
 }
 
-data "aws_subnet_ids" "main" {
+data "aws_subnet_ids" "private" {
   vpc_id = data.aws_vpc.main.id
+  tags = {
+    Tier = "Private"
```

Comment:
In this example we are using the default VPC which does not have the `Tier` tag and only has public subnets. Should we remove the `tags` block and change the name from `privat` to `public`?

Reply:
will update.